lms/static/scripts/frontend_apps/utils/api.js

```diff
@@ -4,7 +4,8 @@
  */
 export class ApiError extends Error {
   constructor(status, data) {
-    const message = data.error_message || data.message || 'API call failed';
```

Comment:
This was causing the actual bug, it was passing either error_message or message to its base class here, but down on line 25 it was only setting `error_message` to `errorMessage` which is the actual value checked in BasicLtiLaunchApp which will determine if it should render the basic Auth dialog or the more verbose dialog.